pigeon.py

```diff
@@ -49,7 +81,8 @@ def decrypt(data):
         )
         return data
 
-    return boto3.client('kms', region).decrypt(CiphertextBlob=b64decode(data))['Plaintext']
+    client = boto3.client('kms', region_name=region)
```

Comment:
The `.client()` function takes a ton of arguments and they're passed in a `**kwargs` kind of way to `boto3.session.Session()`. Making them explicit is a good thing.